project_event/tests/test_project_event_task.py

```diff
@@ -459,7 +472,96 @@ def test_190_main_task_activity_calendar_event_clone_state(self):
             calendar_event = child.get_calendar_event()
             self.assertEqual(calendar_event.state, 'draft')
 
-    def test_200_get_parent_project_id(self):
+    def test_200_compute_actual_total_time(self):
+        # Test case 1: Testing for same actual start and end date
+        date_start_1 = datetime.today()
+        date_end_1 = date_start_1 + timedelta(hours=2)
+        actual_date_start_1 = datetime.today()
```

Comment:
why you use 2 variables if it is the same date?

Reply:
you can use only 1 variable :actual_date_1 = fields.Datetime.to_string(datetime.today())

Reply:
@rimbendhaou : To standardize for different test cases, we handle in that way. I donot think it is problem. Justify me how it improve.